CollisionAlert.lua

```diff
@@ -1,10 +1,12 @@
 
 Instance.properties = properties({
+	{ name="Sensitivity", type="Real", value=0, range={min=0, max=3}, onUpdate="onInstanceUpdate" },
```

Comment:
Since this is a real it should have value and min/max as reals, so 0.0 and 3.0

Reply:
Yep.  I hadn't looked back at the properties after I set the type.